mock/secret_service.go

```diff
@@ -0,0 +1,15 @@
+package mock
+
+import (
+	"context"
+	"errors"
+)
+
+type SecretService struct{}
```

Comment:
This is simple enough that you could just have the mock secret service be a `map[string]string` and implement it.